cmake/common.cmake

```diff
@@ -663,6 +663,10 @@ endfunction()
 
 ## Helper for building end-to-end perf tests using the python infrastucture
 function(add_perf_test)
+  ## Perf tests currently use C++ clients which are not sending HTTP - disable
+  if(HTTP)
+    return()
```

Comment:
Should we write a friendly cmake message for this?

Reply:
Not sure why have this as opposed to if(HTTP) around instances of add_perf_test?